azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -20,6 +20,7 @@
 _SERVICE_ENDPOINT_TYPE = [SERVICE_ENDPOINT_TYPE_GITHUB, SERVICE_ENDPOINT_TYPE_AZURE_RM]
 _SERVICE_ENDPOINT_AUTHORIZATION_SCHEME = [SERVICE_ENDPOINT_AUTHORIZATION_PERSONAL_ACCESS_TOKEN,
                                           SERVICE_ENDPOINT_AUTHORIZATION_SERVICE_PRINCIPAL]
+_ACCESS_LEVELS = ['advanced', 'earlyAdopter', 'express', 'none', 'professional', 'stakeholder']
```

Comment:
Should we keep the input options like.. early-adopter instead of earlyadopter? 
@geverghe  PM choice :)

Reply:
For options, we have stuck to non hyphenated values - not sure if this was intentional but it feels easier to copy paste. For parameters we have followed hyphenated word separation. I would stick to non-hyphenated version for consistency. Are there any other challenges you foresee?

Reply:
No. That sounds good.